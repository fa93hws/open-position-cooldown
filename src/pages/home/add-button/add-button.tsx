import * as React from 'react';
import { Button } from 'rebass';
import { Plus } from '../../../resources/icons';

export const AddButton = () => (
  <Button
    p={2}
    mr={[3, 4]}
    mb={[3, 4]}
    variant="primaryIcon"
    fontSize="0px"
    size="iconMedium"
    sx={{
      position: 'absolute',
      fill: 'white',
      bottom: 0,
      right: 0,
    }}
  >
    <Plus />
  </Button>
);
