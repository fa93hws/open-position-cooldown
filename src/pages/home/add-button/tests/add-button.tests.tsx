import { cleanup } from '@testing-library/react';
import * as React from 'react';

import { AddButton } from '../add-button';

describe('AddButton', () => {
  beforeEach(cleanup);

  it('renders this', () => {
    expect(<AddButton onClick={jest.fn()} />).toMatchRenderedSnapshot();
  });
});
