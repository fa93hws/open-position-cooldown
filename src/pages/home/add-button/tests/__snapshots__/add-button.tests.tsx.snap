// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton renders this 1`] = `
<button
  class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1 MuiButton-containedPrimary"
  tabindex="0"
  type="button"
>
  <span
    class="MuiButton-label"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
      />
    </svg>
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;
